web/src/message_edit.js

```diff
@@ -434,6 +435,7 @@ function edit_message($row, raw_content) {
     condense.hide_message_condenser($row);
     const content_top = $row.find(".message_controls")[0].getBoundingClientRect().top;
 
+    assert(message_lists.current !== undefined);
     const message = message_lists.current.get(rows.id($row));
```

Comment:
Probably moving both of these lines to be the top 2 would be nicer here.

Reply:
done